### ISSUE TITLE
Adding a module to your favoutites doesn't seem to work (development 3.3.0)

### ISSUE THREAD

**Issue by jade-nl** [darktable-org/darktable#6488]
I just wanted to add a module to my favourites and I cannot get it to work.

**To Reproduce**
1. Open an image in darkroom,
2. Go to _Presets -> Manage Presets_,
3. Select the appropriate favourites you have,
4. In the correction group add *defringe* (this works, although the layout needs a bit of work)
5. At this point I'm stuck.

The new module does not show up in the left most *favourites* list and there isn't an *add* button under that  list.

Exiting the *Manage Presets* window, leaving and re-entering darkroom or a full restart doesn't help.

**Expected behavior**
A way to add a module to your favourites.

**Platform**
* darktable        : 3.3.0+1132 g891bb3b9a
* OpenCL installed : Yes (nvidia-opencl-common 450.66-1)
* OpenCL activated : Yes
* Graphics card    : GeForce RTX 2060 SUPER7979 MiB
* Graphics driver  : nvidia (450.66)
* OS               : Linux (kernel 5.7.10-1)
* Distro           : Debian Buster (rel. 10.6)
* Xorg             : 7.7+19

**Additional context**
* Self compiled : Yes.
* Compilers     : gcc 8.3.0 - clang 10.0.1 - LLVM 10.0.1
* Cmake         : 3.13.4
* Gtk           : 3.24.5-1
* Glib          : 2.58.3-2
* Sqlite        : 3.27.2
* Optimization  : None

**Comment by junkyardsparkle** [comment]
Hmm. My "favorites" group has an "add" button underneath it, but since I think I actually deleted my old favorites group and created another one while trying to break the new group manager, I also tried ingesting a 3.2.1 config folder into current master, and going directly to your steps to reproduce... there was still an "add" button under "favorites", both in the "previous config" and "previous config with new layout" presets. So... can't reproduce this one here.
